pages/serverless-functions/faq.mdx

```diff
@@ -190,6 +190,18 @@ There are [several ways to deploy Serverless Functions](/serverless-functions/re
 
 Integration is straightforward. Serverless Functions and Containers can be triggered by events from [Queues](/messaging/concepts/#queues) and [Topics and Events](/messaging/concepts/#topics-and-events), and can easily communicate with services like [Managed Databases](/managed-databases-for-postgresql-and-mysql/) or [Serverless databases](/serverless-sql-databases/). [Serverless Jobs](/serverless-jobs/) can pull data from [Object Storage](/object-storage/), or output processed results into a database. With managed connectors, APIs, and built-in integrations, linking to the broader Scaleway ecosystem is seamless.
 
+### How do Serverless Functions namespaces and Container Registry namespaces interact?
+
+Serverless Functions namespaces and Container Registry namespaces observe the following behaviors:
+
+- Creating a Serverless Functions namespace implicitly creates an empty Container Registry namespace. When a Serverless Function is deployed, the built image will be stored in the Container Registry namespace created.
+
+- Creating a Container Registry namespace **does not** create a Serverless Functions namespace.
+
+- Deploying a function in a Serverless Functions namespace creates an image in the corresponding Container Registry namespace.
+
+- If you delete The Container Registry namespace associated with a Serverless Functions namespace, it will be created again when deploying a function within this Serverless Functions namespace.
```

Comment:
```suggestion
- If you delete the Container Registry namespace associated with a Serverless Functions namespace, it will be created again when deploying a function within this Serverless Functions namespace.
```